src/NzbDrone.Core/Configuration/IConfigService.cs

```diff
@@ -49,6 +49,9 @@ public interface IConfigService
         int MaximumSize { get; set; }
         int MinimumAge { get; set; }
 
+        int ImportListSyncInterval { get; set; }
```

Comment:
This setting isn't valuable to Sonarr. The scheduled task runs every 5 minutes and each list has it's own pre-configured interval, which for Sonarr syncing is the same 5 minute interval, for Plex Watchlist, it's 6 hours.

Reply:
I somehow managed to read the task's interval as 5 hours so thought the setting made sense. I've removed it now